_std/macros/mob_properties.dm

```diff
@@ -178,6 +178,10 @@ To remove:
 #define PROP_DIGESTION_EFFICIENCY(x) x("digestion_efficiency", APPLY_MOB_PROPERTY_PRODUCT, REMOVE_MOB_PROPERTY_PRODUCT)
 #define PROP_CHEM_PURGE(x) x("chem_purging", APPLY_MOB_PROPERTY_SUM, REMOVE_MOB_PROPERTY_SUM)
 #define PROP_REBREATHING(x) x("rebreathing", APPLY_MOB_PROPERTY_SIMPLE, REMOVE_MOB_PROPERTY_SIMPLE)
+#define PROP_TEMP_CHEM_DEPLETION(x) x("chem_deplete_heat", APPLY_MOB_PROPERTY_SIMPLE, REMOVE_MOB_PROPERTY_SIMPLE)
+#define PROP_TEMP_CHEM_EFFECTS(x) x("chem_effect_heat", APPLY_MOB_PROPERTY_SIMPLE, REMOVE_MOB_PROPERTY_SIMPLE)
+#define PROP_TEMP_DIGESTION(x) x("digestion_heat", APPLY_MOB_PROPERTY_SIMPLE, REMOVE_MOB_PROPERTY_SIMPLE)
```

Comment:
doesn't seem to be used?

Reply:
Not yet, I pushed this in a less-than finished state. For instance, lizards immediately die because most of their organs are missing.